.github/workflows/athena_test.yml

```
@@ -60,6 +60,40 @@ jobs:
               fi
+              
+              # For text module, also run real tests including quality drift analysis
+              if [ "$module" = "modules/text/module_text_llm" ]; then
+                echo "Running real tests for $module..."
+                
+                # Install additional dependencies for quality drift analysis
+                $module/.venv/bin/pip install bert-score torch transformers numpy
+                
+                # Download required NLTK data
+                $module/.venv/bin/python -c "import nltk; nltk.download('punkt', quiet=True); nltk.download('punkt_tab', quiet=True)"
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)